src/main/java/org/myrobotlab/vertx/Auction.java

```diff
@@ -0,0 +1,34 @@
+package org.myrobotlab.vertx;
```

Comment:
Everything in this package should be put in a different PR

Reply:
removed